config/utils/buildGrmVoteApiUri.ts

```diff
@@ -0,0 +1,46 @@
+import {
```

Comment:
Can we move this to `components/Vote/api` ? It is a good pattern I believe you introduced that I have been following for a while.

Reply:
I'll do it once everything starts to use VoteWidgetV2. For now, because VoteWidget is a shared component, let's keep this file as a part of utils folder. I'll make sure to move this file when the switch over is done